gef.py

```diff
@@ -6366,7 +6363,7 @@ def tcachebin(tcache_base: int, i: int) -> Tuple[Optional[GlibcChunk], int]:
             err("Incorrect index value, index value must be between 0 and {}-1, given {}".format(GlibcHeapTcachebinsCommand.TCACHE_MAX_BINS, i))
             return None, 0
 
-        tcache_chunk = GlibcChunk(tcache_base)
+        tcache_chunk = GlibcChunk(tcache_base, _type="tcache")
```

Comment:
Something like 

```suggestion
        tcache_chunk = GlibcTcacheChunk(tcache_base)
```

is more explicit